mod-audit-server/src/main/java/org/folio/dao/acquisition/impl/OrderEventsDaoImpl.java

```diff
@@ -0,0 +1,62 @@
+package org.folio.dao.acquisition.impl;
+
+import io.vertx.core.Future;
+import io.vertx.core.Promise;
+import io.vertx.core.json.JsonObject;
+import io.vertx.sqlclient.Row;
+import io.vertx.sqlclient.RowSet;
+import io.vertx.sqlclient.Tuple;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.folio.dao.acquisition.OrderEventsDao;
+import org.folio.rest.jaxrs.model.OrderAuditEvent;
+import org.folio.util.PostgresClientFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Repository;
+
+import static java.lang.String.format;
+
+@Repository
+public class OrderEventsDaoImpl implements OrderEventsDao {
+
+  private static final Logger LOGGER = LogManager.getLogger();
+
+  public static final String TABLE_NAME = "acquisition_order_log";
+
+  private static final String INSERT_SQL = "INSERT INTO %s (id, action, order_id, user_id, user_name, event_date, action_date, modified_content_snapshot) VALUES ($1, $2, $3, $4, $5, $6, $7, $8)";
+
+  @Autowired
+  private final PostgresClientFactory pgClientFactory;
+
+  @Autowired
+  public OrderEventsDaoImpl(PostgresClientFactory pgClientFactory) {
+    this.pgClientFactory = pgClientFactory;
+  }
+
+  @Override
+  public Future<RowSet<Row>> save(OrderAuditEvent orderAuditEvent, String tenantId) {
+    Promise<RowSet<Row>> promise = Promise.promise();
+
+    String query = format(INSERT_SQL, TABLE_NAME);
```

Comment:
do we prepend the tenant id to the table name?

Or a better question is, how do we save data into specific tenant schemas.

Reply:
I think it is determined in this method makeSaveCall: pgClientFactory.createInstance(tenantId)
But interesting, because in other places we specify tenantId.tableName, we will check how its working, thanks for finding